Utils/index.js

```diff
@@ -0,0 +1,10 @@
+import checkproptypes from 'check-prop-types';
+export const findByTestAttr = (component, attr) => {
```

Comment:
actually I expect to see these utils in tests folder, probably makes sense to make them gobal

Reply:
Shifted